src/Investments/index.jsx

```diff
@@ -1,19 +1,31 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { View } from 'react-native';
+import { useDispatch } from 'react-redux';
 import commonStyles from '../common/styles';
 import SearchButton from '../common/HeaderSearchButton';
 import PortfolioValue from './PortfolioValue';
+import { calculatePortfolioValue } from '../redux/investment/actionCreator';
+import PortfolioNews from './PortfolioNews';
 
 const navigationOptions = {
   title: 'Investments',
   headerRight: () => <SearchButton />,
 };
 
-const Investments = () => (
-  <View style={commonStyles.container}>
-    <PortfolioValue />
-  </View>
-);
+const Investments = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(calculatePortfolioValue());
```

Comment:
Now since it is much more than calculating the portfolio value, this name looks outdated. I think we should rename it. Shall we do another branch and pr for that?